[FIX] web_editor: adapt options' icons in RTL

robinlej opened the pull request:
The triangle pointing towards an option in the editor is pointing
right. In a RTL setting, this does not make sense and should be
mirrored.

task-3284274

robodoo commented:
[Pull request status dashboard](https://mergebot.odoo.com/odoo/odoo/pull/120134).

qsm-odoo reviewed (changes requested):
In general, we try to fix bugs in the earliest version we support (14.0). There are exceptions, following common sense depending if the bug is critical or not / easy or not to solve / etc and other devs may have other rules. But for a CSS fix which impacts RTL languages UI, there should not be much debate: 14.0 if the bug is there :+1: 

To retarget a PR, you don't need to rename the branch. Just rebase your branch on the right version, change the target of your PR and push-force. /!\ Before push-forcing or changing the PR target, make sure to make it go back to draft, otherwise there is a small risk to ping the whole company for no reason (there are protection against that now I think but never too careful). I put it back in draft myself here :+1:

robinlej commented:
@qsm-odoo I rebased on 14.0 and changed the base branch.
As discussed, the └ should be fixed in the forward port to 16.0.
I had to add some logic for the carret as the class .o_rtl isn't set on the body before 16.0.

qsm-odoo reviewed (approved):
As discussed, the work will be adapted in further version to handle the └ symbol properly. In this old version, the symbol is part of the translated terms so any problem with it is actually a translation problem (transifex).

@robodoo r+
